app/admin/admin_users.rb

```diff
@@ -72,7 +81,8 @@
   controller do
     # update without password if not provided
     def update_resource(object, attributes)
-      update_method = attributes.first[:password].present? ? :update_attributes : :update_without_password
+      update_method = attributes.first[:password].present? ? :update : :update_without_password
```

Comment:
FYI: 'update_attributes' are deprecated in Rails 6 (will be removed in Rails 6.1) - https://github.com/rails/rails/pull/31998.

Reply:
:+1: